lib/board.rb

```diff
@@ -0,0 +1,53 @@
+class Board
+
+  attr_accessor :cells
+
+  def initialize(cells = nil)
+    @cells = Array.new(9, " ")  
+  end
+
+  def reset!
+    initialize.clear
```

Comment:
This is a little backwards. `initialize` is typically not called directly, only invoked on object instantiation through `new`.